OK 1 of 1 tests passed.
ls.out test 1 failed: diff faulure
